### PR TITLE
Contributing to the website in README

### PR DESCRIPTION
**Pull request by xtuc**
Related to #1001.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,48 @@ $ cd babel.github.io
 * Install `jekyll` dependencies by running `bundle install`
 * `npm i` to install this package's dependencies
 * To start the web server on your machine, check out this repo and run `npm start`.
+
+### Contributing to the website
+
+Babel is composed with multiple packages. Their documentations aren't hosted in this repository.
```
Comment (existentialism) [inline README.md]: Just an idea, but maybe we can reduce this to something like:

>To keep documentation in sync across all of Babel's packages and plugins, the majority of the website is sourced directly from the README files located inside their respective  repositories. ([Babel's packages](https://github.com/babel/babel/tree/master/packages), [Babili's packages](https://github.com/babel/babili/tree/master/packages), etc.).

Thoughts?

File: README.md [hunk 2]
```diff
@@ -21,3 +21,48 @@ $ cd babel.github.io
...
+
+```sh
+git submodule foreach git pull origin master
+```
+
+You can safely push them once updated. Note that the website will during deployement fetch all latest submodule versions.
+
+#### Looking for support?
+
+For questions and support please join our [Slack community](https://slack.babeljs.io/), channel #website.
```
Comment (hzoo) [inline README.md]: Yeah can click to signup + the link

File: README.md (hunk 1 above)
Comment (hzoo) [inline README.md]: Yeah - part of the docs is hosted so it's weird 😛

Comment (xtuc) [inline README.md]: Yes, way better, thanks a lot.

**Comment by hzoo** [comment]
Awesome stuff

**Pull request merged**